bindings/java/mac_run.me

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh -e
+
+cd `dirname $0`
+
+#unset _JAVA_OPTIONS
+if [ "x$JAVA_HOME" = "x" ]; then
+    JAVA_HOME=`java -XshowSettings:properties -version 2>&1 | \
+               awk 'BEGIN { FS="=\\\s*" } /java\.home/ { print $2 }'`
```

Comment:
On Mac the best way to find `JAVA_HOME` is usually with `/usr/libexec/java_home`. I can accept a ` -v` option to request a specific version (e.g. `/usr/libexect/java_home -v 11` for Java 11 (it will return the best match so in my case that points to Java 15).

Reply:
Done